webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path')
```

Comment:
Yeah, probably you're right. 

TBH I've changed to webpack because it's the bundler that I'm used to, but I've isolated the changes in a single commit because I've also questioned myself if it was too much for this use case.

I can remove this commit 👍